src/App.vue

```diff
@@ -314,7 +315,7 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
   get feeSummaryConfirmLabel (): string {
     const isNoFee = this.isChangeFiling || this.isConversionFiling
     if (this.isSummaryMode) {
-      return isNoFee ? 'File Now (No Fee)' : 'File and Pay'
+      return isNoFee && !this.getFilingData.priority ? 'File Now (No Fee)' : 'File and Pay'
```

Comment:
For additional clarity, please wrap the two conditions in parentheses. Thanks!

Explanation: I find ternary expressions like this easy to read/review when they have a simple condition. When there are multiple conditions (or the result is itself an condition) then it's usually better to add parenthese, or break it up into 3 lines, or use if-then-else.

Reply:
👍